### DISCUSSION TITLE
History output

### DISCUSSION THREAD

**Discussion by ahashib** [haiiliin/abqpy#5397] [Q&A]
Hello
I can run my model script successfully.  I have the following line for history output request in my model script. 
mdb.models['Model-1'].HistoryOutputRequest(name='H-Output-top-displacement', createStepName='Step-1', variables=('U2', ), region=regionDef, sectionPoints=DEFAULT, rebar=EXCLUDE)

The problem is when I try to extract the output from the odb script, I get an error. Could anyone help please?
![image](https://github.com/haiiliin/abqpy/assets/151507529/2833cc55-e0b3-432b-a0c4-94d9e30dba81)

**Comment by haiiliin** [comment]
Maybe

```python
history = model.HistoryOutputRequest('F-Output-1', createStepName='Step-1', variables=('U',))
```
